### ISSUE TITLE
Send staging 10-10d submits to Slack

### ISSUE THREAD

**Issue by baltes** [department-of-veterans-affairs/va.gov-team#84619]

**Comment by andrea-merrill** [comment]
@michaelclement same, I think this can be closed, we get notified about failures and it would be too noisy to get notifications for success

**Comment by michaelclement** [comment]
Yeah, I would say this functionality is already covered since we track "missing PEGA status" + submission errors on the staging dashboard anyway, so we're able to tell when things don't make it to S3 staging w/o being on VA network
